lib/Applify.pm

```diff
@@ -616,13 +623,22 @@ See L<Getopt::Long/Options-with-multiple-values> for details.
 
 =item * C<isa>
 
-Specify the class an option should be instantiated as. Example:
+Can be used to either specify a class that the value should be instantiated
+as, or a L<Type::Tiny> object that will be used for coercion and/or type
+validation.
+
+Example using a class:
 
   option file => output => "output file", isa => "Mojo::File";
 
 The C<output()> attribute will then later return an object of L<Mojo::File>,
 instead of just a plain string.
 
+Example using L<Type::Tiny>:
+
+  use Types::Standard "Num";
+  option num => age => "Your age", isa => Num;
```

Comment:
potential nit: Could this example be a little confusing being a Num instead of an Int?

Reply:
Not sure, but I changed it either way.